FlashCardApp/Data/SqlServerDatabaseManager.cs

```diff
@@ -0,0 +1,341 @@
+using System.Configuration;
+using System.Data.SqlClient;
+using FlashCardApp.DTO;
+using FlashCardApp.Models;
+
+namespace FlashCardApp.Data;
+
+public class SqlServerDatabaseManager : IDatabaseManager
+{
+    private string _connectionString = ConfigurationManager.AppSettings.Get("connectionString")!;
+
+    public void CreateStackTable()
+    {
+        using (var connection = new SqlConnection(_connectionString))
+        {
+            using (var command = connection.CreateCommand())
+            {
+                connection.Open();
+
+                command.CommandText = "IF OBJECT_ID(N'Stack', N'U') IS NULL " +
```

Comment:
It's a matter of personal choice (or the convention of your company) but here I'd use @ to create multiline strings and get rid of the + signs. Having said that, even with the plus signs, these sql statements look tidy.